nuxeo-coldstorage/src/main/java/org/nuxeo/coldstorage/helpers/ColdStorageHelper.java

```diff
@@ -119,6 +130,7 @@ public static DocumentModel moveContentToColdStorage(CoreSession session, Docume
         documentModel.addFacet(COLD_STORAGE_FACET_NAME);
         documentModel.setPropertyValue(COLD_STORAGE_CONTENT_PROPERTY, mainContent);
         documentModel.setPropertyValue(FILE_CONTENT_PROPERTY, null);
+
```

Comment:
Could you check your java formatter I am wondering why there is a modification in the `Move` method